man/systemd.network.xml

```diff
@@ -706,6 +706,12 @@
             <citerefentry project='man-pages'><refentrytitle>inet_pton</refentrytitle><manvolnum>3</manvolnum></citerefentry>.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>OnDevice=</varname></term>
+          <listitem>
+            <para>A boolean. Specifies whether to take device as default route.</para>
```

Comment:
I lost network connection :)


On 05/02/2016 02:23 PM, Lennart Poettering wrote:
>
> In man/systemd.network.xml 
> <https://github.com/systemd/systemd/pull/3133#discussion_r61716680>:
>
> > @@ -706,6 +706,12 @@
> >              <citerefentry project='man-pages'><refentrytitle>inet_pton</refentrytitle><manvolnum>3</manvolnum></citerefentry>.</para>
> >            </listitem>
> >          </varlistentry>
> > +        <varlistentry>
> > +          <term><varname>OnDevice=</varname></term>
> > +          <listitem>
> > +            <para>A boolean. Specifies whether to take device as default route.</para>
>
> "everything went down"? What do you mean? Not following?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/systemd/systemd/pull/3133/files/8cabac68a87555d1fc8bced9adb3fc1af42d5159#r61716680>
>